wgpu-hal/Cargo.toml

```diff
@@ -37,7 +37,7 @@ block = { version = "0.1", optional = true }
 foreign-types = { version = "0.3", optional = true }
 
 # backend: Vulkan
-ash = { version = "0.33", optional = true }
+ash = { version = "0.34", optional = true, default-features = false, features = ["debug", "loaded"] }
```

Comment:
Are we able to get away without debug?

Reply:
It would be difficult for little benefit:
- `wgpu_hal::Api` requires many associated types to implement `Debug`
- Because `ash` types wouldn't implement `Debug` we wouldn't be able to derive it so we would have to write manual impl
- `Debug` impls are used here and there for logging
- We would lose quite a bit of information

Reply:
Alright, that seems fine to me, if we're using them, we should keep them.